nss-day-cohort-29/nutshell-spirit-wind
# Messages functionality creation

# Description
This feature 

1. Allows a logged-in user to post a message to the DOM and edit the message. It also incorporates all previous changes from the Master.

2. Allows a logged-in user to log out.

Type of change
New feature (non-breaking change which adds functionality)
Testing Instructions for Change Made
git fetch --all
git checkout messages-functionality-creation
code .
cd src/lib
grunt

input login information, all functionality previously incorporated should appear (including friends), plus messages functionality should appear. Post a few messages as the logged-in user. Edit the message and post the edit. Log out the user, the page should refresh and return to the login screen.

Checklist:
[ ] My code follows the style guidelines of this project
[ ]  I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[ ]  My changes generate no new warnings
[ ]  I have added test instructions that prove my fix is effective or that my feature works
